crates/viewer/re_ui/src/list_item/list_item.rs

```diff
@@ -87,6 +87,9 @@ impl ListItem {
     #[inline]
     pub fn selected(mut self, selected: bool) -> Self {
         self.selected = selected;
+        if selected {
+            self.interactive = true; // Hack needed, or the item is not shown as selected. TODO(emilk): why?
```

Comment:
Well because `interactive` is false, then we `Sense::hover()`, and `interact_selectable` returns the visuals for a noninteractive view.

I think this logic is best handled in `ui()` with some explanation comment.